Project.toml

```diff
@@ -35,7 +35,7 @@ DocStringExtensions = "0.8, 0.9"
 Insolation = "0.8"
 JLD2 = "0.4"
 NCDatasets = "0.11, 0.12, 0.13"
-Plots = "1.39.0"
+Plots = "=1.39.0"
```

Comment:
Gabriele wants to avoid pinning Plots, so I have a separate PR that changes the backend we use for plotting in our tests. I'm trying to merge it early tomorrow

Reply:
https://github.com/CliMA/ClimaCoupler.jl/pull/582

Reply:
Ok sounds great, thanks!